spec/services/prison_offender_manager_service_spec.rb

```diff
@@ -134,7 +139,7 @@
 
         it "raises" do
           expect {
-            described_class.get_pom_at('LEI', 1234)
+            described_class.get_single_pom('LEI', 1234)
```

Comment:
this call is wrong (get_single_pom only takes one argument) so yet again the raise check needs to be more explicit.